src/resources/resource.rs

```diff
@@ -48,9 +48,15 @@ pub(crate) fn find(ctx: &RenderContext) -> Result<HashMap<String, Asset>, Error>
                     continue;
                 }
                 for link in find_assets_in_markdown(&ch.content)? {
-                    let asset = match Url::parse(&link) {
-                        Ok(url) => Asset::from_url(url, &ctx.destination),
-                        Err(_) => Asset::from_local(&link, &src_dir, ch.path.as_ref().unwrap()),
```

Comment:
I was a bit of surprise that test `find_local_asset` got pass and ignore assets outside source directory silently.